configs/openmrs/initializer_config/ampathforms/emergency_department_form.json

```diff
@@ -0,0 +1,461 @@
+                "answers": [
+                  {
+                    "concept": "92eaeb1f-775a-4c18-8029-956c3340cb6d",
+                    "label": "Traumatic injuries"
+                  },
+                  {
+                    "concept": "0c7e2af6-c17f-4387-8884-4c5e1c9d2a17",
+                    "label": "Respiratory"
+                  },
+                  {
+                    "concept": "c9d6f1cc-6d7d-4ef3-9215-8c563acdf5c2",
+                    "label": "Obstetric"
+                  },
+                  {
+                    "concept": "77fdf29f-ff0c-4b70-b500-28353c641e02",
+                    "label": "Musculoskeletal"
+                  },
+                  {
+                    "concept": "48328818-38b0-4cff-9fc4-1d3bef606157",
+                    "label": "Genitourinary"
+                  },
+                  {
+                    "concept": "9a8aad2e-fc07-4da7-9271-b71a31f2b22d",
+                    "label": "Gastrointestinal"
+                  },
+                  {
+                    "concept": "b0cd5e9e-02f2-45b9-bafe-4ddda678c237",
+                    "label": "Cardiovascular"
+                  }
+                ]
+              },
+              "validators": []
+            }
+          ]
+        },
+        {
+          "label": "Medical History",
+          "isExpanded": "true",
+          "questions": [
+            {
+              "label": "Anamnesis",
+              "type": "obs",
+              "required": false,
+              "id": "anamnesis",
+              "questionOptions": {
+                "rendering": "textarea",
+                "concept": "6b8024cb-d2fa-4627-a0b8-5de6e3f6f103"
+              },
+              "validators": []
+            },
+            {
+              "label": "Past medical history",
+              "type": "obs",
+              "required": false,
+              "id": "pastMedicalHistory",
+              "questionOptions": {
+                "rendering": "textarea",
+                "concept": "a5864ffc-d3b7-42fb-bbe7-e4aa080c7bdb"
+              },
+              "validators": []
+            },
+            {
+              "label": "Family history",
+              "type": "obs",
+              "required": false,
+              "id": "familyHistory",
+              "questionOptions": {
+                "rendering": "textarea",
+                "concept": "9aa32cf1-0528-43f8-8f0b-9ec5dfdbdeb4"
+              },
+              "validators": []
+            },
+            {
+              "label": "Previous hospitalization",
+              "type": "obs",
+              "required": false,
+              "id": "previousHospitalization",
+              "questionOptions": {
+                "rendering": "radio",
+                "concept": "a1d46f24-06ff-4f8d-b2b0-e6754bcc37a5",
+                "answers": [
+                  {
+                    "concept": "bf885a9d-9f98-4990-8c20-ac4ace816bfe",
+                    "label": "True"
```

Comment:
Please replace the `True`, `False` with `Yes` and `No` respectively

Reply:
This has been worked on @usamaidrsk